src/js/controllers/addDriver/creatMockData.js

```diff
@@ -0,0 +1,58 @@
+// addDriver 페이지에서
+// db에 보험자 데이터 등록하기 전 목업 데이터 생성하는 모듈
+export default function createMockData(product) {
+  if (product) {
+    // 보험료 납부 기간 (0~120개월)
+    const subsPeriod = Math.floor(Math.random() * 120);
```

Comment:
생각보다 보험료 많이 냈었네요..!! 저는 한 96개월 냈던 것 같았어요 🥲

Reply:
나중에 서울가면 민섭님 차 구경시켜주세요 ㅎㅎ

Reply:
한글 변수명도 좋은 선택지중에 하나입니다!
```suggestion
    const 보험료_납부_기간 = Math.floor(Math.random() * 120);
```

이렇게 변수명을 지으면, 주석도 필요없고 가독성도 가져올 수 있어서 좋은 선택지입니다.

저는 보통 익숙하지 않은 영단어를 활용할 때 한글 변수명을 자주 활용하는 편인 것 같아요